internal/resource/common/attributemapping/attribute_mapping.go

```diff
@@ -34,7 +38,36 @@ func ToSchema() schema.SingleNestedAttribute {
 			"attribute_sources":              attributesources.ToSchema(0, false, true),
 			"issuance_criteria":              issuancecriteria.ToSchema(),
 		},
-		Required:    true,
+		Required:    false,
```

Comment:
Should this be a parameter? This change will affect any other resources using attribute mappings.